pkg/util/validation.go

```diff
@@ -18,19 +18,33 @@ package util
 
 import (
 	"regexp"
-	"strings"
 )
 
-// IsDNSLabel tests for a string that conforms to the definition of a label in
-// DNS (RFC 1123).
-func IsDNSLabel(value string) bool {
-	return IsDNS1123Label(value)
+const kubeChar string = "[A-Za-z0-9]"
+const extendedKubeChar string = "[-A-Za-z0-9_.]"
+const qualifiedToken string = "(" + kubeChar + extendedKubeChar + "*)?" + kubeChar
+
+const LabelValueFmt string = "((" + kubeChar + extendedKubeChar + "*)?" + kubeChar + ")?"
```

Comment:
Why not define this as "(" + qualifiedToken + ")?" - is that offensive to the purist in you?